tests/test_modeling_tf_common.py

```diff
@@ -1393,6 +1393,22 @@ def ids_tensor(shape, vocab_size, rng=None, name=None, dtype=None):
     return output
 
 
+def floats_tensor(shape, scale=1.0, rng=None, name=None):
+    """Creates a random float32 tensor"""
+    if rng is None:
+        rng = random.Random()
+
+    total_dims = 1
+    for dim in shape:
+        total_dims *= dim
+
+    values = []
+    for _ in range(total_dims):
+        values.append(rng.random() * scale)
+
+    return tf.constant(values, shape=shape, dtype=tf.float32)
+
+
```

Comment:
Not sure if this should be added, all other TF models apparently don't need it?

Reply:
Removed. 
c180538